index.js

```diff
@@ -38,7 +38,9 @@ const defaultConfig = {
 };
 
 // Check if Verminal configuration exists in ~/.hyper.js. If not, fall back to default configuration.
-const checkConfig = (config, setting) => (config.hasOwnProperty("verminal") && config.verminal[setting]) || defaultConfig[setting]
+const checkConfig = (config, setting) => config.hasOwnProperty("verminal") && config.verminal[setting] !== undefined ? 
```

Comment:
Can you run Prettier on this file?

Reply:
There is no .prettierrc file. Should I just use the default vscode prettier config?

Reply:
Yeah, please :)

Reply:
Just did using default configuration. Seems like more than my change was formatted though.